src/commands/node.mjs

```diff
@@ -557,36 +639,57 @@ export class NodeCommand extends BaseCommand {
 
     try {
       await tasks.run()
+      self.logger.debug('node start has completed')
     } catch (e) {
       throw new FullstackTestingError(`Error starting node: ${e.message}`, e)
+    } finally {
+      await self.accountManager.close()
     }
 
     return true
   }
 
+  async checkNetworkNodeProxyUp (namespace, nodeId, maxAttempts = 100) {
+    const podArray = await this.k8.getPodsByLabel([`app=haproxy-${nodeId}`, 'fullstack.hedera.com/type=haproxy'])
+
+    if (podArray.length > 0) {
+      const podName = podArray[0].metadata.name
+
+      let attempts = 0
+      while (attempts < maxAttempts) {
+        const logResponse = await this.k8.kubeClient.readNamespacedPodLog(
+          podName, namespace)
+
+        if (logResponse.response.statusCode !== 200) {
+          throw new FullstackTestingError(`Expected pod ${podName} log query to execute successful, but instead got a status of ${logResponse.response.statusCode}`)
+        }
+
+        if (logResponse.body.includes('Server be_servers/server1 is UP')) {
+          return true
+        }
+
+        attempts++
+        this.logger.debug(`Checking for pod ${podName} to realize network node is UP [attempt: ${attempts}/${maxAttempts}]`)
+        await sleep(1000)
+      }
+    } else {
+      throw new FullstackTestingError('TBD')
```

Comment:
TBD -> `proxy for 'node0' is not ACTIVE [ attempt = 50/50 ]`

Reply:
done